src/main/java/blackjack/Application.java

```diff
@@ -37,10 +37,10 @@ private static List<Card> getInitCards(Deck deck) {
     }
```

Comment:
**One**

🔍 One potential performance improvement is in the `Card` class's static initialization block. The `VALUES` set is being computed lazily, which can be beneficial for large datasets. However, if you need to access this set frequently, it might be more efficient to compute and store it once during application startup. 

✨ 훌륭해요! 더 나은 코드를 위해 노력하는 모습이 멋집니다. 😄